changed_from_main.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script determines whether one of the projects in the repo or any of its
```

Comment:
should this really be in the top level directory? 

In fv3core we did use a `.jenkins` folder for these type of things as it is pretty CI related. Or we could put this into a util folder?

Reply:
If our top level directory gets cluttered I could see moving this eventually, but this is an integral part of running the top-level Makefile targets as we're currently planning to use it. If we call it directly in Jenkins instead of calling it inside the Makefile (as I did for test_util), we can move it then.

Reply:
sounds good to me